### ISSUE TITLE
Tabbed widget (Resources)

### ISSUE THREAD

**Issue by terracoda** [phetsims/balloons-and-static-electricity#152]
@jessegreenberg. Here are two recent articles on ARIA Tabbed widgets which came across the WCAG mailing list. They should be helpful for multi-screen sims and how we are handling the multiple screens.
- On Medium, [Danger!Testing Accessibility with real people] (https://medium.com/@LeonieWatson/danger-testing-accessibility-with-real-people-4515f72db648#.k0ng5llrc) 
- On HackPoets, [From HTML to ARIA Tabs, A Travelog] (https://hackpoets.wordpress.com/2016/05/10/from-html-to-aria-tabs-a-travelog/)

The articles provide a code examples (in progress) and also powerful arguments in support of standards-based approaches, and also highlight how the landscape continues to evolve. Progress is happening.

**Comment by jessegreenberg** [comment]
Thanks @terracoda!  I will check this out!

**Comment by jessegreenberg** [comment]
Leonie Watson's article is great, mostly about research and avoiding pitfalls of jumping to conclusions with unsuccessful user testing.  I think that I have fallen into that pitfall as well.  In frustration I have also thought "Perhaps ARIA isn't yet supported enough in the real world to handle this kind of content".

**Comment by jessegreenberg** [comment]
The article from HackPoets should be quite helpful, it has really rich examples of tabs with markup and html.  I will save these links.  Thanks for sending them along @terracoda!

**Comment by jessegreenberg** [comment]
Closing this issue, when we are ready to make multiple screens accessible, an issue in Joist will be created and we will certainly reference these articles.  Thanks @terracoda!
